### DISCUSSION TITLE
Get it to work on Autocode.com

### DISCUSSION THREAD

**Discussion by GoodDay360** [SudhanPlayz/Discord-MusicBot#537] [Ideas]
Can you make a way to deploy it on autocode.com or make your project available there?

**Comment by JotaroKujo0525** [comment]
I don't know if it can. Because me myself don't code or host there. But you can go see it yourself

**Comment by JotaroKujo0525** [comment]
But yeah, I think it can ¯\_(ツ)_/¯

**Comment by GoodDay360** [comment]
Can you make it into a public project there? Since idk anything about nodejs so idk what the difference between a normal VPS host and host in there and I can't seem to find a way to clone the project from Github to there.

**Comment by JotaroKujo0525** [comment]
I mean, you can use other hosts you know ;-;

**Reply by GoodDay360** [reply]
😑Isn't it better if there are more places you can host it...?

**Reply by ItzSudhan** [reply]
> 😑Isn't it better if there are more places you can host it...?

😑 you can litrarly run it anywhere if you know how it works

**Reply by JotaroKujo0525** [reply]
> > 😑Isn't it better if there are more places you can host it...?
> 
> 😑 you can litrarly run it anywhere if you know how it works

Yeah, you can. Is autocode even for hosting though

**Reply by GoodDay360** [reply]
Yes, it is for hosting. I just don't know how to upload it there.

**Reply by JotaroKujo0525** [reply]
Then it's kinda your problem. I don't know how to host there. Get a vps or good hosting perhaps

**Reply by GoodDay360** [reply]
God damn it, I'm broke. 😭😭😥

**Reply by JotaroKujo0525** [reply]
https://repl.it ;-;

**Comment by DarrenOfficial** [comment]
Autocode is for prebuild bot thats made in autocode it self.
